content/terms/implementation-services.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: markdown
+title: Terms of Service for Sourcegraph Implementation Services
+---
+
+Last Modified: May 5, 2023
+
+This document outlines the basic terms and conditions that apply to Sourcegraph's Implementation Services Offerings.
+
+## Implementation Services Terms of Service
+
+### Implementation Services Description
+Dedicated, paid Implementation Services to design, plan, and provide hands-on assistance for Sourcegraph self-hosted production deployments projects. This service will:
```

Comment:
Hands-on (via direct access) or over the shoulder?

Reply:
Over the shoulder. Good Catch.